docs/tutorial/fdcurves.rst

```diff
@@ -9,33 +9,48 @@ The following code loads an HDF5 file and lists all of the FD curves inside of i
 
     import lumicks.pylake as lk
 
-    file = lk.File("example.h5")
-    list(file.fdcurves)  # e.g. shows: "['baseline', '1', '2']"
+    file = lk.File("20200430-192424 FD Curve FD_5_control_forw.h5")
+
+List all the FD curves inside the file::
+
+    >>>list(file.fdcurves)
+    ['FD_5_control_forw']
+
 
 To visualizes an FD curve, you can use the built-in :meth:`.plot_scatter()
 <lumicks.pylake.fdcurve.FdCurve.plot_scatter()>` method::
 
-    # Pick a single FD curve
-    fd = file.fdcurves["baseline"]
+    fd = file.fdcurves["FD_5_control_forw"]
     fd.plot_scatter()
 
+.. image:: figures/fdcurves/fdcurves_scatter.png
+
 Here, :attr:`.fdcurves <lumicks.pylake.File.fdcurves>` is a standard Python dictionary, so we can
 do standard `dict` thing with it. For example, we can iterate over all the FD curve in a file and plot them::
 
     for name, fd in file.fdcurves.items():
         fd.plot_scatter()
-        plt.savefig(name)
+        plt.savefig(name + ".png")
```

Comment:
Shouldn't we mention in the text that we are saving it to a file here?

Personally, I wonder if this saving even adds anything. Might be fine to just omit it.

Reply:
True, I removed the saving part